wcivf/apps/parties/models.py

```diff
@@ -87,6 +87,10 @@ def is_independent(self):
         """
         return self.party_id == "ynmp-party:2"
 
+    @property
+    def is_joint_party(self):
+        return self.party_id == "joint-party:53-119"
```

Comment:
There are more joint parties, and the IDs [always take the format](https://github.com/DemocracyClub/yournextrepresentative/blob/bd0db75e46ef27efb2d2f459087f26f5e03cba21/ynr/apps/parties/importer.py#L41) `joint-party:{}-{}"`

Reply:
So I think `self.party_id.startswith("joint-party:")` would do it?

Reply:
Yes! Makes sense. Change made. 👍